packages/hydra-cli/src/templates/graphql-server/src/index.ts.mst

```diff
@@ -31,11 +31,12 @@ async function bootstrap() {
         endpoint: process.env.GRAPHQL_PLAYGROUND_ENDPOINT || undefined,
         subscriptionEndpoint: process.env.GRAPHQL_PLAYGROUND_SUBSCRIPTION_ENDPOINT || undefined,
         queryTemplates,
+        ...getReqTimeoutMiddleware(Number(process.env.GRAPHQL_SERVER_REQUEST_TIMEOUT) || undefined),
       },
     },
     {
       namingStrategy: new CustomNamingStrategy(),
-      maxQueryExecutionTime: 1000,
+      maxQueryExecutionTime: Number(process.env.GRAPHQL_SERVER_DB_QUERY_TIMEOUT) || undefined,
```

Comment:
Does this actually work? Previously this was set to `1000` which I guess is a second? But it's currently possible to run a query that executes much longer

Reply:
I'm not 100% sure. I'll try to test this but I need to find a gql query that result in a long db query.

But here's why I think it might actually be working, Here's a part of the timeline for the `getProposalsByIds` query ran by the chain setup script (I think the one ran by Pioneer is smaller though):
![getProposalsByIds_partial](https://github.com/Joystream/hydra/assets/6571453/a5a8fe2c-3c76-40ec-9d9b-e09230cf18aa)

If I'm reading this correctly (114 events for 1 graphql.resolve) there's something really wrong with the QN revolvers however the actual DB queries are quick. Even if this is only my localhost I can imagine that in production there might be some un-ending requests which send a massive amount of db queries each taking less than a second to run.